Code/Source/Wrappers/Entity.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020-2021, Robotec.ai sp. z o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <rgl/api/core.h>
+
+namespace RGL::Wrappers
+{
+    class Mesh;
+    class Texture;
+
+    class Entity
```

Comment:
I would at least consider naming this (and Mesh and Texture) a little bit differently. These are names very similar to o3de types and may cause confusion despite the namespace here. Maybe simple RGLEntity, RGLMesh and RGLTexture will do the thing?